src/Symfony/Component/Form/Form.php

```diff
@@ -795,6 +795,27 @@ public function getErrors($deep = false, $flatten = true)
         return new FormErrorIterator($this, $errors);
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @return $this
+     */
+    public function clearErrors(bool $deep = false): self
```

Comment:
@xabbuh I have added a `: self` return type here per your request.  See https://github.com/symfony/symfony/pull/27580#discussion_r196414039 for an explanation of why I didn't also add a return type to the interface.  Let me know your thoughts :)